tests/ami/packages_sanity_tests.py

```diff
@@ -525,9 +534,9 @@ def render_script_template(
 
     template_context["verbose"] = verbose
 
-    template = Template(script_template)
+    env = Environment(loader=FileSystemLoader(SCRIPTS_DIR),)
+    template = env.from_string(script_template)
```

Comment:
Does it also allow us to substitute text in the additional templates (such as `partial/reencode_config_without_bom.ps1.j2`) or is only possible for the main template?

Reply:
I believe nested includes should also be supported, but not 100% :)